scripts/md-gen/hook-commands/main.go

```diff
@@ -4,18 +4,14 @@
 package main
 
 import (
-	"fmt"
 	"os"
-	"path/filepath"
 	"slices"
-	"strings"
 	"time"
 
 	"github.com/juju/charm/v12"
 	"github.com/juju/cmd/v3"
```

Comment:
Will have to watch out on the merge up, this will need changing to the internal package.